data/World/Water Temple MQ.json

```diff
@@ -65,7 +65,7 @@
         "dungeon": "Water Temple",
         "locations": {
             "Water Temple MQ Freestanding Key": "
-                Hover_Boots or can_use(Scarecrow) or logic_water_north_basement_ledge_jump",
+                Hover_Boots or can_use(Scarecrow) or logic_water_north_basement_ledge_jump or can_bonk_crate",
```

Comment:
I'm by no means an expert on logic (and especially not MQ logic), but this change looks suspicious.  Was this `or` meant to be an `and`, possibly grouped with one or more of the other options?

Reply:
yes, should be an and